docs/configuration/files.md

```diff
@@ -57,3 +57,11 @@ configuration files (e.g., user-level configuration will be ignored).
 ## Settings
 
 See the [settings reference](../reference/settings.md) for an enumeration of the available settings.
+
+## Configuring the pip interface
+
+A dedicated [`[uv.pip]`](../reference/settings.md#pip) section is provided for configuring _just_
+the `uv pip` command line interface. Settings in this section will not apply to `uv` commands
+outside of the `uv pip` namespace. However, many of the settings in this section have correllaries
+in the top-level namespace which _do_ apply to the `uv pip` interface unless they are overridden by
+a value in the `uv.pip` section.
```

Comment:
Do you think we should mention _why_ we did this?

Reply:
What would you say?

Reply:
The `uv.pip` settings are designed to adhere more closely to pip's interface, while the `uv` settings use alternate designs in some cases (e.g., `--no-build`).